_i18n/ja/coc.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "QMK 行動規範"
+layout: qmk
+permalink: /ja/coc/
+lang: "ja"
+---
+
+# 行動規範
+
+QMK は、包括的で寛容な、友好的なコミュニティであるよう努めています。私たちは、年齢、障害、民族、性同一性と性表現、経験レベル、国籍、外見、政治的信念、人種、宗教、性的アイデンティティと性向に関係なく誰からの参加でもお待ちしています。
+
+> 「優しい言葉は怒りをそらすが、激しい言葉は怒りをかき立てる。」
+
+私たちのユーザ、貢献者、協力者は、エスカレートして反応するのではなく、お互いに優しさと敬意を持って接し、良い意図と仮定して、可能な場合は穏やかに訂正することが期待されます。私たちの目標はできるだけ正確であることですが、優しさと理解は正しさより価値があります。許容できないふるまいの例には次のものがありますが、これらに限定されません:
```

Comment:
解釈間違っているかもしれませんが、' A rather than B' の A の部分を 'to gently correct' と狭く解釈してみました。

```suggestion
私たちのユーザ、貢献者、協力者は、お互いに優しさと敬意を持って接し、良い意図と仮定して、反発をつのらせるのではなく、出来る限り穏やかに訂正することが期待されます。私たちの目標はできるだけ正確であることですが、優しさと理解は正しさより価値があります。許容できないふるまいの例には次のものがありますが、これらに限定されません:
```